Sources/BraveWallet/Crypto/Onboarding/BackupWalletView.swift

```diff
@@ -34,6 +34,11 @@ struct BackupWalletView: View {
       // user can press return in field to execute when continue button is disabled
       return
     }
+    
+    // password filed resign first responder in case ScrollView gets changed
```

Comment:
nit: hard to understand the actual issue / why this fixes it.
```suggestion
    // Dismiss keyboard for password field before leaving this view #8445.
    // Without dismissal, it's possible the scroll view in the detail view
    // has incorrect frame (sitting above the area the keyboard used to be)
    // which can show the scroll view contents out of the scroll bounds
    // but will block tap interactions.
```